components/automate-ui/src/app/modules/infra-proxy/cookbooks/cookbooks.component.html

```diff
@@ -2,7 +2,11 @@
   <chef-loading-spinner *ngIf="cookbooksListLoading" size="50"></chef-loading-spinner>
   <app-empty-state *ngIf="authFailure" moduleTitle="cookbooks" (resetKeyRedirection)="resetKeyTabRedirection($event)"></app-empty-state>
   <ng-container  *ngIf="!cookbooksListLoading && !authFailure">
-    <chef-table>
+    <div class="empty-section" *ngIf="!cookbooks.length">
+      <img alt="No preview" src="/assets/img/no_preview.gif" />
+      <p>No cookbooks available</p>
+    </div>
+    <chef-table *ngIf="cookbooks.length">
```

Comment:
The html changes on this page--and all the similar pages in this PR--need two unit tests each; in this case:
 * cookbook list displays no_preview icon and hides list when list is empty
 * cookbook list displays list and hides no_preview icon when cookbook is not empty

There are many examples in the code base on how to do that; here is one:
https://github.com/chef/automate/blob/db4a921a16807fd3aeff056dce02ca266edefd35/components/automate-ui/src/app/page-components/client-runs-table/client-runs-table.component.spec.ts#L219-L222

Reply:
@msorens I am getting the below error while running the ```make unit``` command. will try to resolve that error and will update the code.
![Screenshot from 2021-02-22 18-32-11](https://user-images.githubusercontent.com/61003053/108712409-bc259800-753c-11eb-91ea-4ae00b20eb49.png)
![Screenshot from 2021-02-22 18-32-15](https://user-images.githubusercontent.com/61003053/108712415-bdef5b80-753c-11eb-92ae-a563738eacff.png)

Reply:
Have you tried without `sudo` ? I never need to use `sudo` when I run unit tests.

Reply:
yes, I have tried that too still getting the same error

Reply:
In theory, we already have the correct configuration in place for the issue you reported per [this issue](https://github.com/karma-runner/karma-chrome-launcher/issues/158#issuecomment-339265457). You will see that in karma.conf.js.
I also double-checked that the output you are getting up to the error is identical to the output I am getting on my machine, except mine runs successfully to completion.

@tarablack01 wondering if you have any ideas here?

Reply:
yes on my system it got resolved and I have added the changes